lib/checks/visibility/hidden-content.js

```diff
@@ -0,0 +1,17 @@
+if (!axe.commons.dom.isVisible(node)) {
```

Comment:
I think the test should do the following:

1. Look for all nodes that directly have `display:none` or `visibility:hidden`
1. Make sure those nodes have descendants with real content
1. Flag only the node that is causing the hide

The intent is to flag areas on the page that are not being covered by the analysis. `aria-hidden` content does get covered by some of the analysis, so it should not be flagged.